hardhat.config.ts

```diff
@@ -245,9 +243,9 @@ export default {
       "@aave/periphery-v3/contracts/treasury/AaveEcosystemReserveController.sol",
       "@aave/periphery-v3/contracts/adapters/paraswap/ParaSwapLiquiditySwapAdapter.sol",
       "@aave/periphery-v3/contracts/adapters/paraswap/ParaSwapRepayAdapter.sol",
-      "@aave/safety-module/contracts/stake/StakedAave.sol",
-      "@aave/safety-module/contracts/stake/StakedAaveV2.sol",
-      "@aave/safety-module/contracts/proposals/extend-stkaave-distribution/StakedTokenV2Rev3.sol",
+      // "@aave/safety-module/contracts/stake/StakedAave.sol",
```

Comment:
I forked `@aave/safety-module` to bump solidity pragma to ^0.8.0 so that it can compile without legacy flag. But oddly I cannot find this contract? `@aave/safety-module/contracts/proposals/extend-stkaave-distribution/StakedTokenV2Rev3.sol`